src/coreclr/pal/src/exception/machmessage.h

```diff
@@ -189,7 +189,7 @@ class MachMessage
     // The maximum size in bytes of any Mach message we can send or receive. Calculating an exact size for
     // this is non trivial (basically because of the security trailers that Mach appends) but the current
     // value has proven to be more than enough so far.
-    static const size_t kcbMaxMessageSize = 1500;
+    static const size_t kcbMaxMessageSize = 0x1500;
```

Comment:
Would it be better to define this as `sizeof(mach_message_t)` plus some extra space?

Reply:
I've actually found a constant in the headers representing the maximum trailer size, so we can define it as sizeof(mach_message_t) + MAX_TRAILER_SIZE and be safe.

Reply:
Added a commit with that change.

Reply:
Is the MAX_TRAILER_SIZE in a mach source file header? And I am guessing it refers to the security trailers that are mentioned in the comment above?

Reply:
Yes, it is in the Apple headers and it refers to those trailers.